Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.yml

```diff
@@ -0,0 +1,97 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: 'Dragos Worldview'
+  version: -1
+configuration:
+- defaultvalue: https://portal.dragos.com/api/v1/
+  display: Server URL (e.g. https://portal.dragos.com/api/v1/)
+  name: url
```

Comment:
Please use as value here the url base only
for example `https://portal.dragos.com` and in the code use the `api/v1/` as part of the url suffix 
see as reference [Cyberint.py](https://github.com/demisto/content/blob/cecf23cfe31b03b76b63daab80b1b7387139c8b6/Packs/Cyberint/Integrations/Cyberint/Cyberint.py#L89)